features/login_to_account.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Login
+
+    As a user,
+    I would like to navigate to login,
+    In order to login I need to enter my credentials.
+
+Background: 
+Given the following user exist
+| name      | email            | password |
+| Mike Hunt | mike@hotmail.com | password |
```

Comment:
Code formatting is off here. Please install a Gherkin extension to yours VSCode and format code using command + F